library/build.gradle

```diff
@@ -21,6 +21,14 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 22
+
+        def projects = rootProject.getProject().getChildProjects()
+        projects.each {
+            def authority = it.value.android.defaultConfig.manifestPlaceholders.downloadAuthority
```

Comment:
I clearly don't understand how to peruse gradle docs. I never know if it's a groovy or gradle thing